File: 151004/Ivanenko/DC_Lab1/DC_Lab1/DC_Lab1/DTO/TweetRequestTo.cs
using DC_Lab1.DTO.Interface;
using DC_Lab1.Models;

namespace DC_Lab1.DTO
{
    public class TweetRequestTo( int? EditorId, string? Title, string? Content) : IRequestTo
    {
        public int Id { get; set; } = 0;

        public int? EditorId { get; set; } = EditorId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;


    }
}

File: 151001/Shcherba/lab1/src/main/java/by/bsuir/dto/TweetRequestTo.java
package by.bsuir.dto;

import jakarta.validation.constraints.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TweetRequestTo {
    private Long id;
    @NotBlank
    @Size(min = 2, max = 64)
    private String title;
    private String content;
    private Long UserId;
    private Long markerId;
}
